### PR TITLE
feat: Upgraded Nightwatch to 2.0, updated distribued config

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Underlying tools
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

**Overview**
This PR upgrades the Nightwatch version bundled with the `cli-e2e-nightwatch` plugin to the latest v1.6.0.

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
I don't know why `ci/circleci: cli-ui` failed, as the changes are only related to nightwatch plugin

**Comment by vaibhavsingh97** [comment]
@sodatea can you please review, and let me know if there any changes required

**Comment by vaibhavsingh97** [comment]
@sodatea can you please review

File: packages/@vue/cli-plugin-e2e-nightwatch/nightwatch.conf.js [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+          feature_path: 'node_modules/nightwatch/examples/cucumber-js/*/*.feature'
+
+          // start the webdriver session automatically (enabled by default)
+          // auto_start_session: true
+
+          // use parallel execution in Cucumber
+          // parallel: 2 // set number of workers to use (can also be defined in the cli as --parallel 2
+        }
+      }
+    },
```
Comment (sodatea) [inline packages/@vue/cli-plugin-e2e-nightwatch/nightwatch.conf.js:183]: Cucumber-js example is not available in nightwatch 1.7, please remove this.

**Review by sodatea — CHANGES_REQUESTED** [review]

File: packages/@vue/cli-plugin-e2e-nightwatch/nightwatch.conf.js (hunk 1 above)
Comment (sodatea) [reply packages/@vue/cli-plugin-e2e-nightwatch/nightwatch.conf.js:183]: Never mind. I didn't read the title correctly. The problem is that the nightwatch version in the package.json isn't updated, the config is fine.

**Review by sodatea — APPROVED** [review]

**Pull request merged**